plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveMinioDataLake.java

```diff
@@ -127,8 +108,16 @@ public HiveHadoop getHiveHadoop()
 
     @Override
     public void close()
-            throws IOException
+            throws Exception
     {
         stop();
     }
+
+    private enum State
```

Comment:
NIT: Maybe we could move it to BaseTestContainer and make this mechanism test containers wide and as well use enum here ? It's a kind of wrapper to multiple containers.